### PR TITLE
Fix issues in family cards

### LINKED ISSUE #180: Family cards shouldn't have visits
On view for all families

### LINKED ISSUE #181: Family images should just be placeholders

### LINKED ISSUE #183: Family cards should have phone number
nullcheck for family member within a family, every family member has a phone number

### PR DESCRIPTION
**Pull request by alextac98**
Fixes #183  Family cards have phone numbers
Fixes #181  Family image placeholder changed
Fixes #180  Families do not have visits

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,5 @@
 app/schemas/org.fundacionparaguaya.advisorapp.data.local.LocalDatabase/1.json
 app/release
 *.orig
+*.json
+app/schemas/org.fundacionparaguaya.advisorapp.data.local.LocalDatabase/1.json
```
Comment (krconv) [inline .gitignore]: This shouldn't be in gitignore; the schemas file is autogenerated but needed in version control

File: app/src/main/java/org/fundacionparaguaya/advisorapp/adapters/FamiliesAdapter.java [hunk 2]
```diff
@@ -57,7 +57,7 @@ public void onBindViewHolder(FamilyViewHolder holder, int position) {
 
         holder.familyName.setText(family.getName());
 
-        Uri uri = Uri.parse("https://bongmendoza.files.wordpress.com/2012/08/urban-poor-family.jpg");
+        Uri uri = Uri.parse("https://image.ibb.co/gf4qt7/Default_Family.jpg");
```
Comment (krconv) [inline app/src/main/java/org/fundacionparaguaya/advisorapp/adapters/FamiliesAdapter.java]: Can we change this to one of their pictures maybe? I know that we didn't talk about it, but all of the white makes it harder to see the cards

File: app/src/main/java/org/fundacionparaguaya/advisorapp/fragments/FamilyDetailFrag.java [hunk 3]
```diff
@@ -97,7 +99,8 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
             Log.e(FamilyDetailFrag.class.getName(), e.getMessage());
         }
 
-        Uri uri = Uri.parse("https://bongmendoza.files.wordpress.com/2012/08/urban-poor-family.jpg");
+
+        Uri uri = Uri.parse("https://image.ibb.co/gf4qt7/Default_Family.jpg");
```
Comment (krconv) [inline app/src/main/java/org/fundacionparaguaya/advisorapp/fragments/FamilyDetailFrag.java]: Again, maybe this picture: https://s3.us-east-2.amazonaws.com/fp-psp-images/44-3.jpg

File: app/src/main/java/org/fundacionparaguaya/advisorapp/adapters/FamiliesAdapter.java (hunk 2 above)
Comment (krconv) [reply app/src/main/java/org/fundacionparaguaya/advisorapp/adapters/FamiliesAdapter.java]: https://s3.us-east-2.amazonaws.com/fp-psp-images/44-3.jpg

File: app/src/main/java/org/fundacionparaguaya/advisorapp/fragments/FamilyDetailFrag.java [hunk 4]
```diff
@@ -113,6 +116,20 @@ public void onChanged(@Nullable Family family) {
         mFamilyName.setText(family.getName());
         mAddress.setText(family.getAddress());
         mLocation.setText((CharSequence) family.getLocation());
+
```
Comment (krconv) [inline app/src/main/java/org/fundacionparaguaya/advisorapp/fragments/FamilyDetailFrag.java]: It's not as clear what is going on when you use a catch statement for a null pointer;
```java
if (family.getMember() != null) {
    mPhoneNumber.setText(family.getMember().getPhoneNumber());
}
if (family.getAddress() != null) {
    mAddress.setText(family.getAddress());
}
```

File: app/src/main/res/layout/item_allfamilies.xml [hunk 5]
```diff
@@ -1,77 +1,78 @@
...
             android:layout_width="match_parent"
             android:layout_height="match_parent"
-            android:textAppearance="@style/CardHeading"
-            tools:text="Elokda"
             android:layout_marginLeft="15dp"
-            />
+            android:textAppearance="@style/CardHeading"
+            tools:text="Elokda" />
+
         <TextView
```
Comment (krconv) [inline app/src/main/res/layout/item_allfamilies.xml]: Should remove this completely

File: app/src/main/res/layout/item_allfamilies.xml [hunk 6]
```diff
@@ -1,77 +1,78 @@
...
+            android:layout_width="wrap_content"
+            android:layout_height="match_parent"
+            android:layout_alignStart="@+id/family_name"
+            android:layout_marginTop="30dp"
+            android:paddingRight="5dp"
+            android:text="Next Visit:"
+            android:textAppearance="@style/Label"
+            android:visibility="gone"/>
 
         <TextView
```
Comment (krconv) [inline app/src/main/res/layout/item_allfamilies.xml]: Should remove this completely

File: app/src/main/res/layout/item_allfamilies.xml [hunk 7]
```diff
@@ -1,77 +1,78 @@
...
-            android:textAppearance="@style/Label"
-            tools:text="3 Days"
             android:layout_marginTop="30dp"
             android:layout_toRightOf="@+id/next_visit"
-            />
+            android:textAppearance="@style/Label"
+            tools:text="3 Days"
+            android:visibility="gone"/>
 
         <TextView
```
Comment (krconv) [inline app/src/main/res/layout/item_allfamilies.xml]: Should remove this completely

File: app/src/main/res/layout/item_allfamilies.xml [hunk 8]
```diff
@@ -1,77 +1,78 @@
...
-            android:layout_marginTop="60dp"
             android:layout_marginStart="16dp"
+            android:layout_marginTop="60dp"
             android:paddingRight="5dp"
-            />
+            android:text="Last Visit:"
+            android:textAppearance="@style/Body.Small"
+            android:visibility="gone" />
+
         <TextView
```
Comment (krconv) [inline app/src/main/res/layout/item_allfamilies.xml]: Should remove this completely

File: app/src/main/res/layout/item_allfamilies.xml (hunk 7 above)
Comment (alextac98) [reply app/src/main/res/layout/item_allfamilies.xml]: I did't for future implementation

File: app/src/main/java/org/fundacionparaguaya/advisorapp/adapters/FamiliesAdapter.java [hunk 9]
```diff
@@ -57,7 +58,7 @@ public void onBindViewHolder(FamilyViewHolder holder, int position) {
 
         holder.familyName.setText(family.getName());
 
-        Uri uri = Uri.parse("https://bongmendoza.files.wordpress.com/2012/08/urban-poor-family.jpg");
+        Uri uri = Uri.parse("https://s3.us-east-2.amazonaws.com/fp-psp-images/44-3.jpg");
```
Comment (bhylak) [inline app/src/main/java/org/fundacionparaguaya/advisorapp/adapters/FamiliesAdapter.java]: for consistency, this should get the url from the strings file "family_imagePlaceholder" "

**Pull request merged**